### DISCUSSION TITLE
article/notionnext-heo

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#89] [Announcements]
# article/notionnext-heo

这是一个由NotionNext生成的站点

https://docs.tangly1024.com/article/notionnext-heo

**Comment by MrPeterJin** [comment]
您好，首先非常感谢您的主题。想询问一下教程/repo源码中如果要修改 `会循环展示一些技能的图标，可以在 themes/heo/config.js中配置` 的图标的话，这些原始图片应该要放在哪里？repo 里面似乎搜索不到 `/image` 文件夹？

**Comment by jtysfw** [comment]
heo主题修改config.js，vercel报错","，怎么修改部署都不成功。
